src/components/SidePanels/TabPanels/__tests__/utils.test.tsx

```diff
@@ -372,4 +372,41 @@ describe('stateRemovePanelEmptyViews', () => {
     expect(result.panel1.panelViews).not.toHaveLength(0);
     expect(result.panel2.panelViews).not.toHaveLength(0);
   });
+});
+
+describe('splitPanelColumns', () => {
```

Comment:
Could we also add unit test cases for `joinPanelColumns` and `getSnappedHeights` ?

Reply:
yup